content/en/docs/tasks/inject-data-application/define-interdependent-environment-variables.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Define Dependent Environment Variables
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+This page shows how to define dependent environment variables for a container
+in a Kubernetes Pod.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+
+
+
```

Comment:
If you got a chance to send in another commit, pls remove the extra blanks lines like this.

Reply:
done